src/components/SaveGameForm.js

```diff
@@ -124,6 +151,13 @@
                 winner: '',
                 shots:'',
             })
+            setDirtyInputs({
+                location: false,
+                date: false,
+                players: false,
+                winner: false,
+                shots: false,  
+            })
```

Comment:
I thought, instead of writing this, one could also write:
`setDirtyInputs(
    Object.keys(dirtyInputs).forEach((v) => (dirtyInputs[v] = false))
  )`
But it's not working... I dont know why. Never mind..